indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Elements.kt

```diff
@@ -433,11 +440,24 @@ sealed class Asn1Structure(
      * from DER-encoded structures, because this has a chance of altering the structure for non-conforming DER-encoded
      * structures.
      */
-    val isSorted: Boolean = false
+    sortChildren: Boolean,
+
+    /**
+     * Indicates whether this structure should sort their child nodes by default. This is true for SET and for
+     * all custom structure that enforce SET semantics. Note that it is impossible to infer this property correctly when
+     * parsing custom structures. Therefore, it has no impact on [equals].
+     */
+    val shouldBeSorted: Boolean
 ) :
     Asn1Element(tag) {
 
-    val children: List<Asn1Element> = if (!isSorted) children else children.sortedBy { it.tag }
+    val children: List<Asn1Element> = if (!sortChildren) children else children.sortedBy { it.tag }
+
+    /**
+     * indicated whether the structure's children are aurally sorted.
```

Comment:
![image](https://github.com/user-attachments/assets/ba6749cb-b4bf-4ce7-b9e1-601613d6234c)


i think you mean "actually"

Reply:
i do love me some autocorrect!